src/getting_started/changelogs/Changelogs.js

```diff
@@ -8,11 +8,8 @@ export default function Authentication() {
     <section className="Article">
       <h1>Changelogs</h1>
       <section>
-        <p>
-        The API V4 is currently in beta.  There will be regular releases that may
-        introduce breaking changes.  The Developers site contains all changes to
-        API V4 since 2017-09-18.  Please check here regularly for updates.
-        </p>
+        <p>The API is a living system, and as releases are made Changelogs will
```

Comment:
Can you remove "The API is a living system" it sounds really weird. Maybe just say:

As releases are made Changelogs will be published here.